package-lock.json

```diff
@@ -192,6 +192,396 @@
 				"webpack-cli": "^5.0.1"
 			}
 		},
+		"how-to/integrate-with-bloomberg-basic": {
+			"version": "1.0.0",
+			"license": "SEE LICENSE IN LICENSE.MD",
+			"dependencies": {
+				"@openfin/bloomberg": "^1.1.0",
+				"@openfin/core": "^29.73.14",
```

Comment:
Hi Ameet you should delete the main node_modules folder delete package-lock and do a fresh npm install and then npm build from the root. The package lock still has the changes from earlier installs when you were referencing v11.